src/control/controlframepos.cpp

```diff
@@ -0,0 +1,19 @@
+#include "control/controlframepos.h"
+
+#include "moc_controlframepos.cpp"
+
+ControlFramePos::ControlFramePos(const ConfigKey& key, mixxx::audio::FramePos defaultPosition)
+        : ControlObject(key) {
+    setDefaultPosition(defaultPosition);
+    reset();
+    connect(this,
+            &ControlFramePos::valueChanged,
+            this,
+            &ControlFramePos::slotValueChanged,
+            Qt::DirectConnection);
+}
+
+void ControlFramePos::slotValueChanged(double value) {
+    const auto position = mixxx::audio::FramePos::fromEngineSamplePosMaybeInvalid(value);
+    emit positionChanged(position);
```

Comment:
Oh sorry, I have deleted may original comment because I have misread the code. 
The connection is required because we need both signals.